explainer.md

```diff
@@ -72,6 +72,113 @@ There are still many open questions, but a few big ones:
+ * Enable the use of selective disclosure (eg. for more privacy-friendly age 18+ verification)
+ * Enable the PII to be end-to-end encrypted between the wallet application and the relevant verification server
+ * Enable the use of request authentication where a wallet is required to cryptographically verify that the requester has permission to access the credential
+
+Further, this specification aims to improve over the existing communication channels used by websites to talk to wallet apps by:
+ * Enabling browsers and operating systems to provide meaningful credential selection and permission screens to users prior to wallet applications becoming aware of the presentation request
+ * Securely communicate the origin of the requesting site to the wallet application so that it can implement it’s own MITM protection with support of WebPKI
+ * Enabling browsers to inspect requests and provide additional UI affordances to users
+
+> 05.  Do the features in your specification introduce state
+>      that persists across browsing sessions?
+
+No. In particular, the current scope is just about credential presentation (read-only). We have had requests to expand scope to consider credential issuance APIs but no work has begun on that.
+
+> 06.  Do the features in your specification expose information about the
+>      underlying platform to origins?
+
+Not directly. Wallets may (intentionally or inadvertently) expose some information through this API (such as some indication of which wallet app the user is using). But the primary motivation for these wallets over traditional federated authentication systems is that the wallet can act in the user’s interest to protect their privacy even from the credential issuer. Issuers generally choose which wallets to support their credentials in and can impose privacy requirements on those wallets. Users will often have their choice of multiple wallets and are expected to use reputation for privacy as part of their decision making.
+
+> 07.  Does this specification allow an origin to send data to the underlying
+>      platform?
+
+Yes. The credential presentation request is sent to the underlying platform to mediate.
+
+> 08.  Do features in this specification enable access to device sensors?
+
+No
+
+> 09.  Do features in this specification enable new script execution/loading
+>      mechanisms?
+
+No
+
+> 10.  Do features in this specification allow an origin to access other devices?
+
+Not yet, but we expect to expand the API to enable cross-device presentation flows using the same mechanism used by passkeys ([FIDO CTAP](https://fidoalliance.org/specs/fido-v2.2-rd-20230321/fido-client-to-authenticator-protocol-v2.2-rd-20230321.html).
+
+> 11.  Do features in this specification allow an origin some measure of control over
+>      a user agent's native UI?
+
+Not control, no. User agents may add additional affordances for user transparency and control. 
```

Comment:
Are you discounting the underlying Open System as part of the User Agent?

This specific does allow an origin some measure of control over the OS's UI layer, right (e.g. by opening a bottom sheet with a credential selector?)?

Reply:
Agree with @samuelgoto ... further, what fields are requested can have significant influence and could potentially be abused (e.g., an RP asking for 100 things in different ways)